db/oracle_schema.rb

```diff
@@ -140,8 +140,11 @@
     t.string   "base_path"
     t.string   "swagger_version"
     t.boolean  "skip_swagger_validations", limit: nil,                default: false
+    t.integer  "service_id",               limit: 8,   precision: 8
```

Comment:
For oracle it should be only `precision: 38` no?

Reply:
wow you are right @hallelujah ! 👍

Reply:
We are currently talking about this 😄